doc/ci.md

```diff
@@ -20,24 +23,25 @@ These can be automatically created with `./ci/build-scripts/build-schema.sh`.
 
 Checks will run against the codebase in parallel according to the definition inside `./ci/Jenkinsfile` (scripted pipeline).
 
-These tasks will be run inside docker containers. Backend will be checked against a sidecar-container running Postgres.
+These tasks will be run inside docker containers.
+Backend will be checked against a sidecar-container running PostgreSQL.
 
 ### Build
 
 Each and every pull request will be built and tested against a separate CI database.
 
 Steps for all PRs are:
 
-* prepare an empty Postgres database
+* prepare an empty PostgreSQL database
```

Comment:
I am not really sure about this suggestion but do you think that it would hurt to explicitly mention PostGIS since it is an extension of PostgreSQL? Not all PostgreSQL databases have PostGIS installed by default, so specifying this requirement could ensure that anyone setting up the CI/CD pipeline is aware of this dependency.
```suggestion
* prepare an empty PostgreSQL database with the PostGIS extension installed
```

Reply:
PostgreSQL is the official name, the original fix was simply about that. But I like your suggestion.